packages/core/stories/status-indicator.doc.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import { StatusIndicator } from "@jpmorganchase/uitk-core";
+import { CharacteristicUsage } from "css-variable-docgen-components";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Core/StatusIndicator"
+  component={StatusIndicator}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Status Indicator
+
+A status indicator can be used on its own or within another component (e.g. `Tooltip`) to help convey a message.
+
+<Canvas>
+  <Story id="core-statusindicator--default-status-indicator" />
+</Canvas>
+
+## How you can use status indicator
+
+Use this component by passing in your chosen `status` from the available options.
+
+You can also customize the indicator's `size`.
+
+## Configuring status indicator
+
+### API
+
+```
+import { StatusIndicator } from "@jpmorganchase/uitk-core";
+```
+
+### Props
+
+You can set the indicator status and other configurtions via its props.
+
+<ArgsTable of={StatusIndicator} />
+
+### Anatomy
+
+<Canvas>
+  <ComponentAnatomy>
+    <StatusIndicator status="info" />
+  </ComponentAnatomy>
+</Canvas>
+
+## Features
+
+### Status indicators
+
+The status indicator offers four severity levels that set a distinctive indicator and color.
+
+<Canvas>
+  <Story id="core-statusindicator--all-status-indicators" />
+</Canvas>
+
+### Use when...
+
+An `error` indicator should be used when there is critical information preventing an action.
+
+A `warning` or `info` (depending on the severity) indicator should be used when there is important, but non blocking, information to be conveyed to the user.
+
+Use a `success` indicator to confirm that an action has been completed.
+
+## Characteristics Used
+
+<CharacteristicUsage of={StatusIndicator} />
+
+## Related components
+
+The status indicator component is based on the following component:
+
+- [icon](?path=/story/documentation-styles-and-theming-foundations-iconography--page)
```

Comment:
Is there a reason icon isn't capitalized?

Reply:
It should be